### PR TITLE
Added new file in locations and updated the README.md

### PR DESCRIPTION
**Pull request by sorkan**
### What was wrong?
client requested new site to be added to the locations

### How was it fixed?
Setup a new file for the location

#### Cute Animal Picture

> put a cute animal picture here.

Added new location for The DosaHut.

### CODE REVIEW

File: locations/The_DosaHut/The_DosaHut.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# The Dosa Hut
+
+* Address: 440 US Highway 130 Ste 10 , East Windsor, NJ 08520-2787
+* Hours: Open 24 hours a day.
+* Phone: (877 )-252-3672
```
Comment (craftninja) [inline locations/The_DosaHut/The_DosaHut.md]: Maybe remove the space after the area code?

File: locations/The_DosaHut/The_DosaHut.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# The Dosa Hut
+
+* Address: 440 US Highway 130 Ste 10 , East Windsor, NJ 08520-2787
```
Comment (craftninja) [inline locations/The_DosaHut/The_DosaHut.md]: Maybe remove space after 10 and before comma?

File: README.md [hunk 3]
```diff
@@ -13,6 +13,7 @@ This repository has places to eat for the following cities:
 * [Orlando, FL](/locations/Orlando)
 * [The Woodlands, TX](/locations/The Woodlands)
 * [Vernon Hills, IL](/locations/Vernon Hills)
+* [The DosaHut, NJ](/locations/The_DosaHut)
```
Comment (craftninja) [inline README.md]: This link should be to the city

**Comment by sorkan** [comment]
Changed name of location file and in README.md file

**Pull request merged**